website/content/docs/enterprise/license/faq.mdx

```diff
@@ -20,6 +20,13 @@ This check is part of the server boot-up process.
 In previous versions of HashiCorp enterprise products, one server could distribute a license to other servers via the Raft protocol.
 This will no longer work since each server must be able to find a valid license during the startup process.
 
+## Q: What are the upgrade requirements?
+
+All customers on Consul Enterprise 1.8/1.9 must first upgrade their agents to the latest patch release.
+The license file must also be set in an environment variable or file path, otherwise the Consul agents will fail to retrieve the license with a valid agent token.
```

Comment:
```suggestion
All customers on Consul Enterprise 1.8/1.9 must first upgrade their client and server agents to the latest patch release.
During the upgrade the license file must also be configured on client agents in an environment variable or file path, otherwise the Consul agents will fail to retrieve the license with a valid agent token.
```